src/model/quotes.js

```diff
@@ -1102,6 +1126,31 @@ class QuotesModel {
     return this.removeEmptyKeys(ret)
   }
 
+  /**
+   * Generates and returns an object containing API spec compliant lowercase HTTP request headers for JWS Signing
+   *
+   * @returns {object}
+   */
+  generateRequestHeadersForJWS (headers, noAccept) {
+    const ret = {
+      'Content-Type': 'application/vnd.interoperability.quotes+json;version=1.0',
+      date: headers.date,
+      'fspiop-source': headers['fspiop-source'],
+      'fspiop-destination': headers['fspiop-destination'],
+      'fspiop-http-method': headers['fspiop-http-method'],
+      'fspiop-signature': headers['fspiop-signature'],
+      'fspiop-uri': headers['fspiop-uri'],
+      'User-Agent': null, // yuck! node-fetch INSISTS on sending a user-agent header!? infuriating!
```

Comment:
I do not think this is applicable since I see that axios is being used?

Reply:
removed